src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -142,9 +142,9 @@ class Comment extends React.Component<Props, State> {
         const isMenuVisible = (canDelete || canEdit) && !isPending;
 
         return (
-            <div className="bcs-comment-container">
+            <div className="bcs-Comment">
                 <Media
-                    className={classNames('bcs-comment', {
+                    className={classNames('bcs-Comment-content', {
```

Comment:
Minor, but how about `bcs-Comment-media`?

Reply:
Maybe; I'll think about in the next PR extracting the tasks code.